src/core/types/chains.ts

```diff
@@ -36,19 +39,70 @@ export const hardhatOptimism: Chain = {
   testnet: true,
 };
 
+export const chainHoleski: Chain = {
```

Comment:
for normalization we might want to use `Holesky` with `y` here

Reply:
oh lol will change

Reply:
about the new chains, we can add them wen ready